internal/repos/phabricator.go

```diff
@@ -10,6 +10,7 @@ import (
 	"github.com/inconshreveable/log15"
 
 	"github.com/sourcegraph/sourcegraph/internal/api"
+	"github.com/sourcegraph/sourcegraph/internal/api/internal_api"
```

Comment:
I think I personally would prefer 
```suggestion
	internalapi "github.com/sourcegraph/sourcegraph/internal/api/internal"
```

but no strong preference

Reply:
I think naming it `internal` will prevent packages outside of `internal/api` from importing this, no?

Reply:
oh lord 🙈 yeah probably. I think I am mainly concerned about the `_` not so much the repetitiveness in the path :)

Reply:
OH hey it works!

Reply:
Oh nevermind it doesn't:

```
enterprise/internal/codemonitors/email/email.go:12:2: use of internal package github.com/sourcegraph/sourcegraph/internal/api/internal not allowed (typecheck)
        internalapi "github.com/sourcegraph/sourcegraph/internal/api/internal"
```

Reply:
I couldn't find another package in the repo with an underscore name, so I renamed to `/internalapi` for consistency 5780bbed5a2bbb686ee795983acf7136039f3d68